src/FSharpLint.Core/Rules/Conventions/SourceLength/SourceLengthHelper.fs

```diff
@@ -4,23 +4,90 @@ open System
 open FSharpLint.Framework
 open FSharpLint.Framework.Suggestion
 open FSharp.Compiler.Text
+open FSharpLint.Framework.ExpressionUtilities
+open System.Text.RegularExpressions
```

Comment:
@webwarrior-ws please move System.* to proper place

Reply:
done